packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts

```diff
@@ -301,12 +302,72 @@ export const getStyles = (props: ISliderStyleProps): ISliderStyles => {
             transform: 'translateY(-6px)'
           }
     ],
+    regularLabel: [
+      classNames.regularLabel,
+      {
+        position: 'absolute',
+        height: '0px',
+        background: theme.palette.neutralTertiaryAlt,
+        selectors: {
+          [HighContrastSelector]: {
+            backgroundColor: 'WindowText'
+          }
+        }
+      },
+      {
+        transform: 'translateY(6px) translateX(-6px)'
+      },
+      vertical && {
+        margin: '0 auto',
+        marginBottom: '20px',
```

Comment:
This margin can be combined as `margin: '0 auto 20px'`

Reply:
done